src/analysis/retail/monk/mistweaver/modules/spells/ManaTea.tsx

```diff
@@ -105,7 +106,9 @@ class ManaTea extends Analyzer {
       overhealing: 0,
       stacksConsumed: getManaTeaStacksConsumed(event),
       manaRestored: this.manaRestoredSinceLastApply,
-      channelTime: getManaTeaChannelDuration(event),
+      channelTime: event?.prepull
+        ? 0.5 * getManaTeaStacksConsumed(event)
```

Comment:
It's a hasted channel and energizing brew would affect this so this won't quite be accurate, might need a more complicated estimation function.

Reply:
done